.github/workflows/ci.yaml

```diff
@@ -12,30 +12,30 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
```suggestion
    runs-on: ubuntu-latest
```

Reply:
`ubuntu-latest` is still 20.04, I'd prefer defining the actual version
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources